pkg/testing/fixture.go

```diff
@@ -235,6 +258,74 @@ func ExtractArtifact(l Logger, artifactFile, outputDir string) error {
 	return nil
 }
 
+// Runs the given beat for a specified period of time. Will fail if an error appears in the logs.
+// Will refuse to run if the given binary is elastic-agent
+func (f *Fixture) RunBeat(ctx context.Context, runFor time.Duration) error {
```

Comment:
Do you really need to duplicate `Run` into `RunBeat`? Could `Run` just be adjusted possibly with the options like `f.Run(ctx, testing.WithArgs("..."))`?

Reply:
ugh, yah, you're right, I don't particularly like it either, now that I think about it.